app/Http/Controllers/LockerController.php

```diff
@@ -133,6 +114,7 @@ public function unlock(string $lockerGuid, LockerUnlockRequest $request)
 
         return response()->json([
             'message' => 'OK.',
+            'claimId' => $lockerClaim->id ,
```

Comment:
Data should be put in a `data` field. Instead of just showing `claimId` (which should be `claim_id`), isn't it better to show other claim data as well?

In that case, use `new LockerClaimResource($lockerClaim)`.